### PR TITLE
templates: fix windows test

### PR DESCRIPTION
**Pull request by ksimon1**
this PR fixes check if win vms are up, adds retry logic, disables win2k19 test

### CODE REVIEW

File: automation/test-windows.sh [hunk 1]
```diff
@@ -123,7 +123,7 @@ for size in ${sizes[@]}; do
     set +e
     current_time=0
     # run ipconfig /all command on windows vm
-    while [[ $(_oc exec -it winrmcli -- ./usr/bin/winrm-cli -hostname $ipAddressVMI -port 5985 -username "Administrator" -password "Heslo123" "ipconfig /all") != *"$ipAddressVMI"* ]] ; do 
+    while [[ $($(_oc exec -it winrmcli -- ./usr/bin/winrm-cli -hostname $ipAddressVMI -port 5985 -username "Administrator" -password "Heslo123" "ipconfig /all") | grep "IPv4 Address") != *"$ipAddressVMI"* ]] ; do
```
Comment (fromanirh) [inline automation/test-windows.sh]: I'm not sure you need the double shell exec (nested $()) here. Couldn't you just use a pipe?

Comment (MarSik) [reply automation/test-windows.sh]: The nesting is actually wrong, it will first generate new command "output of winrm | grep" and execute it

Comment (ksimon1) [reply automation/test-windows.sh]: fixed

**Comment by fromanirh** [comment]
Disabling win2019 test is unfortunate, but since the failures seems to be related only to that lane, we can't stop the world while we troubleshoot the issue, especially since we are looking to add libvirt/qemu settings recommended from long time (aka not experimental or risky)

**Comment by fromanirh** [comment]
this improves CI, so I think we should have it in.

**Pull request merged**